android/app/src/main/java/app/organicmaps/search/SearchFragment.java

```diff
@@ -383,16 +397,17 @@ private List<HiddenCommand> getHiddenCommands()
   void showSingleResultOnMap(@NonNull SearchResult result, int resultIndex)
   {
     final String query = getQuery();
-    SearchRecents.add(query, requireContext());
+    if (Config.isSearchHistoryEnabled())
+      SearchRecents.add(query, requireContext());
     SearchEngine.INSTANCE.cancel();
     SearchEngine.INSTANCE.setQuery(query);
 
     if (RoutingController.get().isWaitingPoiPick())
     {
       SearchResult.Description description = result.description;
       String subtitle = description != null
-          ? Utils.getLocalizedFeatureType(requireContext(), description.featureType)
-          : "";
+                        ? Utils.getLocalizedFeatureType(requireContext(), description.featureType)
```

Comment:
Please remove all the unnecessary code formatting from the commit here and in other places.

Reply:
@biodranik  I believe this part of the code is not related to me

Reply:
Of course, it's your IDE that formatted the code. Please revert these changes back.